src/coreclr/jit/valuenum.cpp

```diff
@@ -1672,6 +1677,27 @@ ValueNumStore::Chunk::Chunk(CompAllocator alloc, ValueNum* pNextBaseVN, var_type
                     // Since this value is always the same, we represent it as a static.
                     m_defs = &s_specialRefConsts[0];
                     break; // Nothing to do.
+
+#if defined(FEATURE_SIMD)
+                case TYP_SIMD8:
+                {
+                    m_defs = new (alloc) Alloc<TYP_SIMD8>::Type[ChunkSize];
+                    break;
+                }
+
+                case TYP_SIMD16:
+                {
+                    m_defs = new (alloc) Alloc<TYP_SIMD16>::Type[ChunkSize];
+                    break;
+                }
+
+                case TYP_SIMD32:
+                {
+                    m_defs = new (alloc) Alloc<TYP_SIMD32>::Type[ChunkSize];
+                    break;
+                }
+#endif // FEATURE_SIMD
+
```

Comment:
As we discussed with Tanner on Discord, we need a dedicated `TYP_SIMD12` maps/chunks as well, because in HIR we don't treat SIMD16/SIMD12 equivalently.

We also discussed that we'd _like_ to treat them equivalently, but that should be a separate (larger) effort.

Reply:
Fixed.